config/yugabyte/demo/featurebench_colocated.yaml

```diff
@@ -1,5 +1,5 @@
 type: YUGABYTE
-createdb: create database cyugabyte with colocated=true;
+createdb: DROP database if exists yb1;CREATE database yb1 WITH colocated = true;
```

Comment:
Does this work? I tried this locally `DROP database if exists cyugabyte; create database cyugabyte with colocated=true;` and got the error: `com.yugabyte.util.PSQLException: FATAL: database "if" does not exist`

Reply:
Ignore, upon delete the local repo and cloning again, this worked. Not sure what the problem was earlier.